.github/workflows/_merge_pr.yml

```diff
@@ -0,0 +1,11 @@
+on:
```

Comment:
It is best not to use reusable workflows because it is not to be reused.

Reply:
fixed! thank you!